Learn: What is a URL: edits (#33031)
mdn/content: files/en-us/learn/common_questions/web_mechanics/what_is_a_url/index.md

## Summary

> **URL** stands for _Uniform Resource Locator_. A URL is nothing more than the address of a given unique resource on the Web. Alongside {{Glossary("Hypertext")}} and the {{Glossary("HTTP")}} protocol, **_URL_** is one of the key concepts of the Web as we know it. It is also one of the key mechanisms used by users of {{Glossary("Browser","browsers")}} to retrieve any published resource (such resources can be HTML pages, CSS documents, images, etc) on the web.

Reviewer: Thanks. I'm not a fan of this page at all. However if we're going to improve it, let's say a lot less.

```suggestion
A **URL** (Uniform Resource Locator) is the address of a unique resource on the internet. It is one of the key mechanisms used by {{Glossary("Browser","browsers")}} to retrieve published resources, such as HTML pages, CSS documents, images, and son on.
```
